packages/flutter/lib/src/material/ink_highlight.dart

```diff
@@ -83,23 +83,23 @@ class InkHighlight extends InteractiveInkFeature {
   late AnimationController _alphaController;
 
   /// Whether this part of the material is being visually emphasized.
-  bool get active => _active;
-  bool _active = true;
+  bool get showing => _showing;
+  bool _showing = true;
```

Comment:
why this rename?

Reply:
(i ask only because it is potentially breaking, so if there's not a compelling reason to do it, it might be worth not bothering)